api/src/main/resources/hibernate.cfg.xml

```diff
@@ -80,6 +80,7 @@
 		<mapping resource="org/openmrs/api/db/hibernate/Cohort.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/CohortMembership.hbm.xml"/>
 		<mapping resource="org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml" />
+		<mapping resource="org/openmrs/api/db/hibernate/ProviderRole.hbm.xml" />
```

Comment:
Did you intend to add an entry for the provider role?

Reply:
It should not appear anywhere in your changes because it already exists in the upstream code base. If the pull request merge conflicts get you into a mess, sometimes it is easier to simply discard all your changes and do them again with a totally new pull request.